File: BPMOPMobile_VietNamAirLines/BPMOPMobile_VietNamAirLines/BPMOPMobileV1.iOS/Model/ViewRow.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

using Foundation;
using UIKit;

namespace BPMOPMobileV1.Model
{
    public class ViewRow: ViewBase
    {
        public int RowType { get; set; }
        public List<ViewElement> Elements { get; set; }
    }
}

File: VNABPMOP/app/src/main/java/com/vuthao/bpmop/base/model/dynamic/ViewRow.java
package com.vuthao.bpmop.base.model.dynamic;

import java.util.ArrayList;

public class ViewRow extends ViewBase {
    private int RowType;
    private ArrayList<ViewElement> Elements;

    public ViewRow() {
    }

    public ViewRow(int rowType, ArrayList<ViewElement> elements) {
        RowType = rowType;
        Elements = elements;
    }

    public int getRowType() {
        return RowType;
    }

    public void setRowType(int rowType) {
        RowType = rowType;
    }

    public ArrayList<ViewElement> getElements() {
        return Elements;
    }

    public void setElements(ArrayList<ViewElement> elements) {
        Elements = elements;
    }
}
